apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/_components/Sidebar/CommitSelector/NewDraftCommitModal/index.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  Button,
+  CloseTrigger,
+  FormWrapper,
+  Input,
+  Modal,
+  TextArea,
+  useCurrentProject,
+} from '@latitude-data/web-ui'
+import { createDraftCommitAction } from '$/actions/commits/create'
+import { ROUTES } from '$/services/routes'
+import useCommits from '$/stores/commitsStore'
+import { useRouter } from 'next/navigation'
+import { useServerAction } from 'zsa-react'
+
+export default function NewDraftCommitModal({
+  open,
+  setOpen,
+}: {
+  open: boolean
+  setOpen: (open: boolean) => void
+}) {
+  const { data, mutate } = useCommits()
+  const { project } = useCurrentProject()
+  const router = useRouter()
+  const { error, executeFormAction, input } = useServerAction(
```

Comment:
@geclos This is the new version of `zsa-react` Look how nicely it works with the form. The new `input` field gives me the values present in the form. I'm not convinced we should always use our hook instead of this one. For simple forms like this works super nice